.github/workflows/mkosi.yml

```diff
@@ -105,7 +105,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
-      - uses: systemd/mkosi@63fc1fde5b1aac1abf07ac499068c2b62263dafb
+      - uses: systemd/mkosi@2d338abea5770801c2c1afa8dbb829a5cba975dc
```

Comment:
Looks like 63fc1fde5b1aac1abf07ac499068c2b62263dafb (combined with 2dc99fdadb) was the latest commit where Fedora 40 images didn't fail to build on Fedora 40 hosts. With the main branches those images can't be built:
```sh
$ python3 -Werror -Xdev -m mkosi -E MESON_OPTIONS=--werror -E SANITIZERS=address,undefined -E MESON_VERBOSE=1 -E LLVM=1  -T --release=40 --force --force -a
...
>>> Running trigger-install scriptlet: glibc-common-0:2.39-22.fc40.x86_64
>>> Stop trigger-install scriptlet: glibc-common-0:2.39-22.fc40.x86_64
‣  Running prepare script /home/vagrant/systemd/mkosi.images/build/mkosi.conf.d/centos-fedora/mkosi.prepare…
spec not found at pkg/fedora/systemd.spec, run mkosi with -ff to make sure the spec is cloned
‣ "/work/prepare final" returned non-zero exit code 1.
...

Reply:
@evverx https://github.com/systemd/mkosi/pull/2980 will probably fix your issue

Reply:
Looks like now it fails with
```sh
...
>>> Stop trigger-install scriptlet: glibc-common-0:2.39-22.fc40.x86_64
‣  Running prepare script /home/vagrant/systemd/mkosi.images/build/mkosi.conf.d/centos-fedora/mkosi.prepare…
/scripts/mkosi-chroot: line 2: /home/vagrant/VENV/bin/python3: No such file or directory
/scripts/dnf: line 5: /home/vagrant/VENV/bin/python3: No such file or directory
```
I'll double-check.

Reply:
@evverx That's expected, mkosi's sandbox is now implemented using python and we don't make the venv's interpreter available in the sandbox. I suppose I can add a fallback to system python if the same interpreter can't be found but I don't know if this would be a satisfying solution for you.

Reply:
> a fallback to system python if the same interpreter can't be found in the sandbox

I think it should probably work most of the time but if the interpreters are different it can probably lead to unexpected glitches every now and then. Another option would probably be to fail loudly if it's run in virtual environments (and it should probably be documented that mkosi can't be run there and the system installation should be used instead)